templates/cron-upload.sh.j2

```diff
@@ -15,14 +18,10 @@ restic backup \
 
 echo "...done"
 
-echo "
-## Restic forget ##
-"
+title "Restic forget"
 restic forget --keep-last 5 --prune
 echo "...done"
 
-echo "
-## Restic check ##
-"
+title "Restic check"
 restic check
-echo "...done"
+echo "...done"
```

Comment:
Do you mind adding a `\n` at the end of this line?

Reply:
done. I will take care of this from here on.